index.html

```diff
@@ -1901,36 +1901,38 @@ <h1>Greetings Huckleberry!</h1>
         Set4 #278;; 4; Momentum Builder;
         Set4 #279;; 4; Rotorcycle;
 
-        Set4 #306;; 4FS; Wyatt, Junk Collector;		
+        Set4 #306;; 4FS; Wyatt, Junk Collector;
 		Set4 #307;; 4TTJJ; Genetrix Irel IV;
 
 		Set1004 #1;; 1F; Signal Flare;
-		Set1004 #2;; 2FF; Cauldron Cookbook; 
+		Set1004 #2;; 2FF; Cauldron Cookbook;
 		Set1004 #3;; 4F; Arms Enthusiast;
 		Set1004 #4;; 6FFF; Waystone Titan;
 		Set1004 #5;; 1T; Auralian Restorationist;
-		Set1004 #6;; 2T; Tome of Repetition; 		
+		Set1004 #6;; 2T; Tome of Repetition;
 		Set1004 #7;; 5TT; Gravewatch Guardian;
 		Set1004 #8;; 7TT; Sea of Teeth;
 		Set1004 #9;; 2J; Unwavering Exorcist;
 		Set1004 #10;; 4JJ; Xulta Convoy;
 		Set1004 #11;; 5JJ; Dead or Alive;
 		Set1004 #12;; 7JJ;  Telut, Queen's Hand;
 		Set1004 #13;; 1P; Messenger Hawk;
-		Set1004 #14;;  3P; Trumpeting Warhowler; 
-		Set1004 #15;; 4PP; Poaching Drake; 
-		Set1004 #16;; 6P; Maul; 
-		Set1004 #17;; 0S; Last Chance; 
-		Set1004 #18;; 1S; Shakedown; 
-		Set1004 #19;; 4SS; Vara, Vengeance-Seeker; 
+		Set1004 #14;;  3P; Trumpeting Warhowler;
+		Set1004 #15;; 4PP; Poaching Drake;
+		Set1004 #16;; 6P; Maul;
+		Set1004 #17;; 0S; Last Chance;
+		Set1004 #18;; 1S; Shakedown;
+		Set1004 #19;; 4SS; Vara, Vengeance-Seeker;
 		Set1004 #20;; 6SS; Eremot, the Gathering Dark;
 		Set1004 #21;; 8TTSS; Azindel, Revealed;
-		Set1004 #22;; 8FJP; Enemy of My Enemy; 
+		Set1004 #22;; 8FJP; Enemy of My Enemy;
 		Set1004 #23; 1X;; Sealed Writ; power, undepleted
-		
+
 		Set4 #308;;  5FP; Tamarys, the Geomancer;
 		Set4 #309;; 6; Caiphus, Wandering King;
+    Set4 #310;; 4FFF; Yushkov, the Usurper;
```

Comment:
Let's keep the indentation consistent for now. Atom is probably defaulting to spaces instead of tabs which I agree with normally, but this file was created using tabs for indentation.
```suggestion
		Set4 #310;; 4FFF; Yushkov, the Usurper;
```